zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/metrics/MetricsTags.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.broker.exporter.metrics;
+
+/** Collection of tags used in micrometer metrics */
+public class MetricsTags {
```

Comment:
:upside_down_face: can be `final` and declare a private empty constructor to prevent useless instantiation.

Reply:
:wrench: This mixes two different types of tags: the generic ones set on the registry (partitionId, exporterId) and ones specific to the `MetricsExporter` (jobs, type). This, and that this class is defined in the package used for the `MetricsExporter` is a bit confusing IMO. Instead of splitting this class accordingly into two which each define just two constants, I'd just go back to inlining the label names again.